src/components/Calculator.jsx

```diff
@@ -2,6 +2,7 @@ import { useState } from 'react';
 import Result from './Result';
 import Keys from './keys';
 import calculate from '../logic/calculate';
+import Quote from './Ouote';
 
```

Comment:
- [Optional] Dear @Paul-tes, I believe you have misspelled the `Quote`  component into `Ouote`. As professional programmers, we need to be curious about our naming standards. I would suggest naming your component `Quotes.jsx`.

Reply:
thanks